src/User32.Desktop/User32.cs

```diff
@@ -156,6 +177,12 @@ public static partial class User32
         [return: MarshalAs(UnmanagedType.Bool)]
         public static extern bool MessageBeep(MessageBeepType uType);
 
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern MessageBoxResult MessageBox(IntPtr hWnd, string text, string caption, MessageBoxOptions options);
+
+        [DllImport(nameof(User32), SetLastError = true)]
```

Comment:
Are you sure this will work? I cannot find documentation about getting extended error information if this function fails.